flink-clients/src/test/java/org/apache/flink/client/program/ClientConnectionTest.java

```diff
@@ -115,13 +115,13 @@ public void run() {
 
 			try {
 				// wait until the caller is successful, for at most the given time
-				long now = System.currentTimeMillis();
-				long deadline = now + MAX_DELAY;
+				long now = System.nanoTime();
+				long deadline = now + MAX_DELAY*1000000;
```

Comment:
agree, although I commonly use 1000_000 in fact:) I planed to use [this detailed idea style](https://raw.githubusercontent.com/dawidwys/flink/bee981be8aa83225a8e3a736700eae445d79cece/tools/FlinkCodeStyle.xml) after digging today's discussions in mail list. Is this OK?:)